hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/Route.java

```diff
@@ -85,19 +100,26 @@
     @SuppressWarnings("unchecked")
 	@Override
     public void configure() {
-
+    	  	
     	init();
 		setupSSLContextPharmanetRegistry(getContext());
 
 		String pharmaNetUrl = String.format(pharmanetUri + "?bridgeEndpoint=true&sslContextParameters=#%s&authMethod=Basic&authUsername=%s&authPassword=%s", SSL_CONTEXT_PHARMANET, pharmanetUser, pharmanetPassword);
 		log.info("Using pharmaNetUrl: " + pharmaNetUrl);
```

Comment:
I've asked @daveb-hni to fix this logging as we shouldn't be logging usernames/passwords.

Reply:
Removed this logging and logging of other sensitive data for JMB and HIBC url